decidim-assemblies/spec/presenters/decidim/resource_locator_presenter_spec.rb

```diff
@@ -0,0 +1,148 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  describe ResourceLocatorPresenter, type: :helper do
+    let(:organization) { create(:organization, host: "1.lvh.me") }
+
+    let(:route_fragment) { "assemblies/#{participatory_space.slug}" }
+    let(:admin_route_fragment) { "assemblies/#{participatory_space.slug}" }
+
+    let(:participatory_space) do
+      create(:assembly, organization:)
+    end
+
+    let(:component) do
+      create(:component, id: 1, participatory_space:)
+    end
+
+    let(:resource) do
+      create(:dummy_resource, id: 1, component:)
+    end
+
+    context "with a component resource" do
+      describe "#url" do
+        subject { described_class.new(resource).url }
+
+        it { is_expected.to eq("http://1.lvh.me:#{Capybara.server_port}/#{route_fragment}/f/1/dummy_resources/1") }
+
+        context "when specific port configured" do
+          before do
+            allow(ActionMailer::Base)
+              .to receive(:default_url_options)
+              .and_return(port: 3000)
+          end
+
+          it { is_expected.to eq("http://1.lvh.me:3000/#{route_fragment}/f/1/dummy_resources/1") }
+        end
+      end
+
+      describe "#path" do
+        subject { described_class.new(resource).path }
+
+        it { is_expected.to eq("/#{route_fragment}/f/1/dummy_resources/1") }
+      end
+
+      describe "#show" do
+        subject { described_class.new(participatory_space).show }
+
+        it { is_expected.to start_with("/admin/#{admin_route_fragment}") }
```

Comment:
As the URL that origintates the bug is actually with a GET parameter, this example passes currently in develop without any other change. We need to change it from start_with to equals, so we should have detected this bug 

```suggestion
        it { is_expected.to eq("/admin/#{admin_route_fragment}") }
```